mozilla-release/config/version_win.pl

```diff
@@ -113,7 +113,7 @@ sub getNextEntry
 
 my $bufferstr="    ";
 
-my $MILESTONE_FILE = "$topsrcdir/config/milestone.txt";
+my $MILESTONE_FILE = "$topsrcdir/browser/config/version_display.txt";
```

Comment:
Yes, we can replace with new name, but then we can face a problem on merge with next major FF version. It is better to have somekind of "code style" document how we will process such kind of issues. In this case this is not only renaming in one place - if i change name here I must change it everywhere in script (and potentially get a lot of merge conflicts in future)